qemu/tests/balloon_stress.py

```diff
@@ -38,15 +43,19 @@ def run_video():
     timeout = float(params.get("login_timeout", 360))
     session = vm.wait_for_login(timeout=timeout)
 
-    driver_name = params["driver_name"]
-    utils_test.qemu.setup_win_driver_verifier(driver_name, vm, timeout)
+    if params['os_type'] == 'windows':
+        driver_name = params["driver_name"]
+        utils_test.qemu.setup_win_driver_verifier(driver_name, vm, timeout)
 
```

Comment:
should be removed.

Reply:
@yanan-fu I think it didn't be removed because we must check whether driver enable on win.

Reply:
IMHO, It's better to leave the blank line here for clear logical code block division, Thanks!

Reply:
@ybduan  ok, it is just my habit, it is up to you.